components/omega/src/ocn/Eos.h

```diff
@@ -0,0 +1,257 @@
+#ifndef OMEGA_EOS_H
+#define OMEGA_EOS_H
+//===-- ocn/Eos.h - Equation of State --------------------*- C++ -*-===//
+//
+/// \file
+/// \brief Contains functors for calculating specific volume
+///
+/// This header defines functors to be called by the time-stepping scheme
+/// to calculate the specific volume based on the choice of EOS
+//
+//===----------------------------------------------------------------------===//
+
+#include "AuxiliaryState.h"
+#include "Config.h"
+#include "EosConstants.h"
+#include "HorzMesh.h"
+#include "MachEnv.h"
+#include "OmegaKokkos.h"
+#include "TimeMgr.h"
+#include <string>
+
+namespace OMEGA {
+
+enum class EosType {
+   Linear,       /// Linear equation of state
+   TEOS10Poly75t /// Roquet et al. 2015 75 term expansion
+};
+
+//
+/// TEOS10 75-term polynomial
+class TEOS10Poly75t {
+ public:
+   Array2DReal specVolPcoeffs;
+
+   /// constructor declaration
+   TEOS10Poly75t(int NVertLevels);
+
+   KOKKOS_FUNCTION void operator()(const Array2DReal &SpecVol, I4 ICell,
+                                   I4 KChunk,
+                                   const Array2DReal &ConservativeTemperature,
```

Comment:
ConservativeTemperature and AbsoluteSalinity are kinda long. I think it's ok to use ConservTemp and AbsSalinity (or even AbsSalt).

Reply:
I second this suggestion